pkg/test/framework/components/galley/kube.go

```diff
@@ -113,7 +171,14 @@ func (c *kubeComponent) ApplyConfigOrFail(t *testing.T, ns namespace.Instance, y
 
 // DeleteConfig implements Galley.DeleteConfig.
 func (c *kubeComponent) DeleteConfig(ns namespace.Instance, yamlText ...string) (err error) {
-	panic("NYI: DeleteConfig")
+	for _, txt := range yamlText {
```

Comment:
shouldn't we be removing them from resources as well?

Reply:
Not sure how we can model that. The resources are files generated for kubectl use, there is nothing that allows correlation of yamlText with those files in this context.

In either case, the deletes are idempotent, so there is no side-effect to not removing them from the files.

Reply:
I already do something like this for deletes in the native environment. You can parse the first part of the resource to find its kind, name, namespace, etc. Then you can maintain a map of the resources by that information.